Modules/arraymodule.c

```diff
@@ -32,18 +35,28 @@ static struct PyModuleDef arraymodule;
 struct arraydescr {
     char typecode;
     int itemsize;
-    PyObject * (*getitem)(struct arrayobject *, Py_ssize_t);
-    int (*setitem)(struct arrayobject *, Py_ssize_t, PyObject *);
+    PyObject * (*getitem)(char *, Py_ssize_t);
+    int (*setitem)(char *, Py_ssize_t, PyObject *);
     int (*compareitems)(const void *, const void *, Py_ssize_t);
     const char *formats;
     int is_integer_type;
     int is_signed;
 };
 
+#ifdef _MSC_VER
+#define Py_ALIGN_AS(x) __declspec(align(x))
+#else
+#define Py_ALIGN_AS(x) _Alignas(x)
```

Comment:
`_Py_ALIGN_AS` is now defined in `pymacro.h`; remove the `#ifdef Py_GIL_DISABLED` there to use it the regular build.

Reply:
Done.